test/e2e/Makefile

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# If you update this file, please follow:
+# https://suva.sh/posts/well-documented-makefiles/
+
+# Use GOPROXY environment variable if set
+
+.DEFAULT_GOAL:=help
+
+# Which architecture to build - see $(ALL_ARCH) for options.
+# if the 'local' rule is being run, detect the ARCH from 'go env'
+# if it wasn't specified by the caller.
+local : ARCH ?= $(shell go env GOOS)-$(shell go env GOARCH)
+ARCH ?= linux-amd64
+platform_temp = $(subst -, ,$(ARCH))
+GOOS = $(word 1, $(platform_temp))
+GOARCH = $(word 2, $(platform_temp))
+
+GOPROXY := $(shell go env GOPROXY)
+ifeq ($(GOPROXY),)
+GOPROXY := https://proxy.golang.org
+endif
+export GOPROXY
+
+REPO_ROOT := $(shell git rev-parse --show-toplevel)
+
+help:  ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-25s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+## --------------------------------------
+## Binaries
+## --------------------------------------
+
+TOOLS_DIR := $(REPO_ROOT)/hack/tools
+BIN_DIR := bin
+TOOLS_BIN_DIR := $(TOOLS_DIR)/$(BIN_DIR)
+GINKGO := $(GOPATH)/bin/ginkgo
+KUSTOMIZE := $(TOOLS_BIN_DIR)/kustomize
+OUTPUT_DIR := _output/$(GOOS)/$(GOARCH)/bin
+GINKGO_FOCUS ?=
+
+.PHONY:ginkgo
+ginkgo: # Make sure ginkgo is in $GOPATH/bin
+	go get github.com/onsi/ginkgo/ginkgo
+
+.PHONY: run
+run: ginkgo
```

Comment:
suggest adding `-cloudplatform=$(CLOUDPLATFORM)`

and setting `CLOUDPLATFORM ?= kind`

Reply:
Done, I also updated the README